docs/configuration/tracing.md

```diff
@@ -22,6 +22,12 @@ Træfik supports two backends: Jaeger and Zipkin.
   # Default: "traefik"
   #
   serviceName = "traefik"
+    
+  # Span name limit allows for name truncation in case of very long Frontend/Backend names
```

Comment:
Can we append something like

"This can prevent certain tracing providers from dropping such traces."

?

Similar below.